service/payment_test.go

```diff
@@ -144,6 +145,43 @@ func TestUnitCreatePaymentSession(t *testing.T) {
 		So(w.Code, ShouldEqual, 200)
 	})
 
+	cfg.PaymentServiceURL = "https://payments.companieshouse.gov.uk/payments/"
+	cfg.PaymentServicePath = "/pay"
+
+	Convey("Valid response body", t, func() {
```

Comment:
Instead of adding a new test, add steps to the preceding test to avoid duplication.